build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/projects/e/config/rush-project.json

```diff
@@ -1,10 +1,16 @@
 {
+  "$schema": "../../../../../../../libraries/rush-lib/src/schemas/rush-project.schema.json",
   "operationSettings": [
     {
       "operationName": "_phase:build",
       "outputFolderNames": ["dist"],
+      "allowCobuildWithoutCache": true,
+      "disableBuildCacheForOperation": true,
       "sharding": {
-        "count": 75
+        "count": 75,
+        "shardOperationSettings": {
```

Comment:
Isn't it implied that all of the options in `sharding` are "shardOperationSettings"?

Reply:
Similar to the below, this one's a little confusing but I wanted some way to specify operations for both the collate operation and the shard operation. This felt like a good place, as it gets passed as the operation settings when we create the new operation in https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/logic/operations/ShardedPhaseOperationPlugin.ts#L177-L181

Reply:
There's definitely a case that shards shouldn't be included in this `allowCobuildWithoutBuildCache` experiment, but it definitely makes it easier to test with.

Reply:
addressed in https://github.com/microsoft/rushstack/pull/4923